### PR TITLE
skip operator if claim file couldn't be parsed

### PR DESCRIPTION
**Pull request by edcdavid**
build-depends: https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/91

### CODE REVIEW

File: run-basic-batch-operators-test.sh [hunk 1]
```diff
@@ -311,6 +311,29 @@ while IFS=, read -r package_name catalog; do
 	cleanup
 	waitDeleteNamespace "$ns"
 
+	# Check parsing claim file
+	./tnf claim show csv -c "$reportDir"/claim.json -n "$package_name" -t "$CNF_TYPE" "$addHeaders" || status=$?
+
+	# if parsing claim file fails, skip this operator
+	if [ "$status" != 0 ]; then
+		# Add per operator links
+		{
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:320]: No need for status.

```suggestion
	./tnf claim show csv -c "$reportDir"/claim.json -n "$package_name" -t "$CNF_TYPE" "$addHeaders" || {
	# if parsing claim file fails, skip this operator
		# Add per operator links
		{
```

File: run-basic-batch-operators-test.sh [hunk 2]
```diff
@@ -311,6 +311,29 @@ while IFS=, read -r package_name catalog; do
...
+			# Add tnf_config link
+			echo ", tnf_config: "
+			echo '<a href="/'"$REPORT_FOLDER_RELATIVE"'/'"$package_name"'/tnf_config.yml">'"link"'</a>'
+
+			# New line
+			echo "<br>"
+		} >>"$REPORT_FOLDER"/"$INDEX_FILE"
+
+		cleanup
+
```
Comment (rdavid) [inline run-basic-batch-operators-test.sh:333]:
```suggestion
		cleanup
```

File: run-basic-batch-operators-test.sh (hunk 1 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh]: fixed

File: run-basic-batch-operators-test.sh (hunk 2 above)
Comment (edcdavid) [reply run-basic-batch-operators-test.sh:340]: fixed

File: run-basic-batch-operators-test.sh [hunk 3]
```diff
@@ -311,6 +311,29 @@ while IFS=, read -r package_name catalog; do
...
+			# Add error message
+			echo "Results for: <b>$package_name</b>, "'<span style="color: red;">Operator installation failed due to tasty internal error, skipping test</span>'
+
+			# Add tnf_config link
+			echo ", tnf_config: "
+			echo '<a href="/'"$REPORT_FOLDER_RELATIVE"'/'"$package_name"'/tnf_config.yml">'"link"'</a>'
+
+			# New line
+			echo "<br>"
+		} >>"$REPORT_FOLDER"/"$INDEX_FILE"
```
[the remark this answers is not in the document]
Comment (edcdavid) [reply run-basic-batch-operators-test.sh]: fixed

**Comment by ramperher** [comment]
The issue is in our side, sorry for the noise, I've created this change to fix that: https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/91

**Review by greyerof — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**